README.md

```diff
@@ -1,2 +1,54 @@
-# Railsment
-An educational Ruby on Rails web application for traking payments
+ # Railsment
+
+Railicious is an educational Rails application for tracking payments.
+
+![screenshot](./screenshot.png)
+
+Live demo available on [heroku](https://railsment.herokuapp.com/).
```

Comment:
- [ ] Please make sure your app is working correctly on the live demo link because it's crashing the second I try to register 🙏

Reply:
It is working now. I don't know what was the problem.  
But that was issue on Heroku side not my code.